qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -3062,10 +3063,16 @@ def _load_unload():
 
         self.assertTrue(triton_client.is_server_live())
         self.assertTrue(triton_client.is_server_ready())
-        self.assertTrue(
-            load_before_unload_finish[0],
-            "The test case did not replicate a load while async unloading. Consider increase concurrency.",
-        )
+
+        # This test can replicate a load while async unloading on machines with
+        # sufficient concurrency. Regardless on whether it is replicated or not,
+        # the server must not crash.
```

Comment:
WAR LGTM, but for this test as a whole, I think it would be good to leave in a `# TODO: Consider revisiting this test` comment or similar, as I think it's a bit of a strange test due to hand-wavy conditions.

Reply:
Sure. [Add some TODOs for the sub-test](https://github.com/triton-inference-server/server/pull/6762/commits/5cb234e12b4e6a03b87468d3efea5f92c1e4a272)